activities/value-and-impact/policy-documents.md

```diff
@@ -7,6 +7,7 @@ explains: where we are mentioned in public policy documents
 
 ## 2022
 
+* Ministry of the Interior and Kingdom Relations of  Netherlands: [Open source working: beyond the non-binding approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```

Comment:
Should this read "the Netherlands"

Reply:
Google for some reason renders it without in its info sidebar, but I can't find any official or unofficial doc that also does that. I'll add it back.